storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -181,7 +198,7 @@ export const RadioGroup: Story = {
         <Radio aria-required="true" invalid={args.invalid} name="about" value="ander_bedrijf">
           Ander soort bedrijf
         </Radio>
-        <Radio aria-required="true" invalid={args.invalid} name="about" value="evenement">
+        <Radio aria-required="true" checked invalid={args.invalid} name="about" value="evenement">
```

Comment:
I don't think we should show radio buttons that are checked by default, it's an anti-pattern.